docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
+| **volume-nocopy**     | Disables propagating the volume by copying files from the container's filesystem to the volume. By default, if an empty volume is attached to a container, the volume is propagated with the files and directories that are present at the mount-path (`dst`) inside the container. Setting this option disables that behavior, and mounts the volume, even if it's empty. This option takes an optional value, and can be either `true` (or `1`), or `false` (or `0`). If this option is provided without a value, then `true` is used as a default (`volume-nocopy=true` is equivalent to `volume-nocopy`).<br /><br />Note that docker does *not* copy files from the container's filesystem for bind-mounts (`type=bind`). Bind-mounts always use the files/directories from the host.
+| **volume-opt**        | Volume driver-specific options to use when creating the volume. Options are provided as comma-separated list of key/value pairs, for example, `volume-opt=some-option=some-value,some-other-option=some-other-value`. For available options, refer to the documentation of the volume driver that is used.
+
+#### Differences between "--mount" and "--volume"
+
+The `--mount` flag features most options that are supported by the `-v` /
+`--volume` flag for `docker run`. There are some differences;
+
+- The `--mount` flag allows specifying a volume driver, and volume driver
+  options *per volume*, without having to create volumes in advance. When using
+  `docker run`, only a single volume driver can be specified (using the
+  `--volume-driver` flag), which is shared by all volumes.
+- The `--mount` flag allows specifying custom metadata ("labels") for the volume,
+  without having to create the volume out of band.
+- When using `type=bind`, the host-path must refer to an *existing* path on the
+  host, and is not automatically created if the path does not exist. If the
+  specified path does not exist on the host, an error is produced, and the
+  service will fail to be deployed succesfully.
+- The `--mount` flag does not allow you to relabel volumes with `Z` or `z`
+
+#### Important considerations when using volumes and bind-mounts
+
+
+
+### Examples
+
+**Create a service using a named volume**
+
+The following example creates a service that uses a named volume:
+
+```bash
+$ docker service create \
+  --name my-service \
+  --replicas 3 \
+  --mount type=volume,source=my-volume,destination=/path/in/container,volume-label="color=red",volume-label="shape=round" \
+  nginx:alpine
+```
+
+For each replica of the service, docker requests a volume named "my-volume"
+from the default ("local") volume driver on the host that the task is deployed
+on. If the volume does not exist on the host that the task is deployed on, a
+new volume is created, and the "color" and "shape" labels are set on the volume.
```

Comment:
nit: Favor active 

The Engine creates a new volume and applies the color and shape labels to the volume.